src/CloudNative.CloudEvents/MimeUtilities.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2021 Cloud Native Foundation.
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Text;
+
+namespace CloudNative.CloudEvents
+{
+    // TODO: Consider this name and namespace carefully. It really does need to be public, as all the event formatters are elsewhere.
+    // But it's not ideal...
+
+    /// <summary>
+    /// Utility and extension methods around MIME.
+    /// </summary>
+    public static class MimeUtilities
+    {
+        // TODO: Should we return null, and force the caller to do the appropriate defaulting?
+        /// <summary>
+        /// Returns an encoding from a content type, defaulting to UTF-8.
+        /// </summary>
+        /// <param name="contentType">The content type, or null if no content type is known.</param>
+        /// <returns>An encoding suitable for the charset specified in <paramref name="contentType"/>,
+        /// or UTF-8 if no charset has been specified.</returns>
+        public static Encoding GetEncoding(this ContentType contentType) =>
```

Comment:
UTF-8 as a default seems okay.

Reply:
I think it depends on the context. For most of *our* contexts, it's probably okay. I'd still like to leave the TODO there for now, but I'm kinda hoping we won't find anywhere that it's wrong :)